src/info/host.rs

```diff
@@ -0,0 +1,45 @@
+use crate::errors;
+use crate::mlua;
+use crate::Inject;
+
+use mlua::prelude::*;
+
+use std::fs::read_to_string;
+
+#[derive(Clone, Debug)]
+pub(crate) struct Host {
+    pub model: String,
+    pub manufacturer: String,
+}
+
+impl Host {
+    pub fn new() -> Self {
```

Comment:
This should accept `k: &Kernel` and return `Option<Self>`.

The body of the function should then be wrapped in
```rs
match k.name.as_str() {
  "Linux" => { /* your code */ }
  _ => None
}
```
That way, on platforms other than linux, the fetch doesn't crash.

Reply:
Makes sense!